Assets/Scripts/Grid.cs

```diff
@@ -850,7 +850,7 @@ private void Tile_Changed(TileData.TileType newType)
         //If a type is changed to Unoccupied, then add 1 energy
         if (newType == TileData.TileType.Unoccupied)
         {
-            energyCounter.AddEnergy(1);
+            energyCounter.AddEnergy();

```

Comment:
energyPerCell should be supplied as the argument at this method call rather than being hardcoded to replace the amount in the AddEnergy method.